specification/monitor/resource-manager/Microsoft.Monitor/stable/2023-04-03/monitoringAccounts_API.json

```diff
@@ -0,0 +1,561 @@
+          ]
+        },
+        "internalId": {
+          "description": "An internal identifier for the metrics container. Only to be used by the system",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "IngestionSettings": {
+      "description": "Settings for data ingestion",
+      "type": "object",
+      "properties": {
+        "dataCollectionRuleResourceId": {
+          "description": "The Azure resource Id of the default data collection rule for this workspace.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "dataCollectionEndpointResourceId": {
+          "description": "The Azure resource Id of the default data collection endpoint for this workspace.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "AzureMonitorWorkspace": {
+      "description": "Properties of an Azure Monitor workspace",
+      "type": "object",
+      "properties": {
+        "accountId": {
+          "description": "The immutable ID of the Azure Monitor workspace. This property is read-only.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "metrics": {
+          "description": "Information about metrics for the Azure Monitor workspace",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Metrics"
+            }
+          ],
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the Azure Monitor workspace. Set to Succeeded if everything is healthy.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Deleting",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "defaultIngestionSettings": {
+          "description": "The Data Collection Rule and Endpoint used for ingestion by default.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/IngestionSettings"
+            }
+          ],
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "privateEndpointConnections": {
+          "description": "List of private endpoint connections",
+          "type": "array",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnection"
```

Comment:
There is another swagger in your new tag uses another version of `PrivateEndpointConnection`: https://github.com/Azure/azure-rest-api-specs/blob/09319ca4735368425620c4c85f084e3c01944428/specification/monitor/resource-manager/Microsoft.Insights/preview/2021-07-01-preview/privateLinkScopes_API.json#L461. Is it possible to consolidate them? It will cause SDK generating two models for this.

Reply:
Then if microsoft.insights/privatelinkscopes update to the v4 in the future new version, this will be an SDK breaking change.

Reply:
I don't think that team has any plans to upgrade to v4, and even if they did, wouldn't they need to implement a new api-version, the existing api-version would still be v2.  

How do you recommend with proceed?

Reply:
Let me add all languages' SDK owner to discuss. @ArcturusZhang @MaryGao @qingquanzhao @msyyc

Reply:
Approved for Go breaking changes. We'll resolve all duplicate schema problem in the next release using directives.